devops/provision/Dockerfile

```diff
@@ -24,10 +24,7 @@ RUN set -ex \
  && git clone --branch develop https://github.com/WildbookOrg/wbia-tpl-pyhesaff.git \
  && git clone --branch develop https://github.com/WildbookOrg/wbia-tpl-pyflann.git \
  && git clone --branch develop https://github.com/WildbookOrg/wbia-tpl-pydarknet.git \
- && git clone --branch develop https://github.com/WildbookOrg/wbia-tpl-pyrf.git \
- # Depricated
- && git clone --branch develop https://github.com/WildbookOrg/wbia-deprecate-tpl-brambox \
- && git clone --branch develop https://github.com/WildbookOrg/wbia-deprecate-tpl-lightnet
```

Comment:
Let's keep checking out `wbia-deprecate-tpl-lightnet` similar to `wbia-utool`, even though it exists on PyPI

Reply:
Ok I removed the last commit